File: Python/003_Program/program.py
#!/usr/bin/env python3
# Directory Lister - List files and directories
import os

def main():
    print("=== Directory Lister ===")
    path = input("Enter directory path: ")
    
    if os.path.isdir(path):
        print(f"\nContents of {path}:")
        print("\nFiles:")
        for item in os.listdir(path):
            full_path = os.path.join(path, item)
            if os.path.isfile(full_path):
                size = os.path.getsize(full_path)
                print(f"  {item} - {size} bytes")
        
        print("\nDirectories:")
        for item in os.listdir(path):
            full_path = os.path.join(path, item)
            if os.path.isdir(full_path):
                print(f"  {item}/")
    else:
        print("Directory not found!")

if __name__ == "__main__":
    main()

File: CSharp/003_Program/Program.cs
// Directory Lister
using System;
using System.IO;
class Program {
    static void Main() {
        Console.Write("Directory path: ");
        string path = Console.ReadLine();
        if (Directory.Exists(path)) {
            Console.WriteLine("\nFiles:");
            foreach (var file in Directory.GetFiles(path))
                Console.WriteLine($"  {Path.GetFileName(file)}");
            Console.WriteLine("\nDirectories:");
            foreach (var dir in Directory.GetDirectories(path))
                Console.WriteLine($"  {Path.GetFileName(dir)}");
        }
    }
}